src/librustdoc/html/render/write_shared.rs

```diff
@@ -778,7 +782,9 @@ impl TraitAliasPart {
             }
             path.push(format!("{remote_item_type}.{}.js", remote_path[remote_path.len() - 1]));
 
-            let part = OrderedJson::array_sorted(
+            implementors.sort_unstable();
+
+            let part = OrderedJson::array_unsorted(
```

Comment:
This is definitely an improvement, as it eliminates an allocation, but we're still allocating an intermediate string for each element.

`OrderedJson` could be *much* more efficient if it had a method that appended using `serde_json::to_writer`.

I can open an issue for improving the `OrderedJson` interface, if you want.